parsers/ES.py

```diff
@@ -177,17 +177,17 @@ def fetch_production(
                 # NOTE the LNG terminals are not built yet, so power generated by "gas" or "combined" in ES-CN domain is actually using oil.
                 # Recheck this every 6 months and move to gas key if there has been a change.
                 # Last checked: 2022-06-27
+                MAPPING = PRODUCTION_MAPPING
```

Comment:
```suggestion
                fuel_mapping = PRODUCTION_MAPPING
```
Since the value might change it's not a constant, so lower case keys should be used.